projects/storefrontlib/src/cms-components/checkout/services/express-checkout.service.ts

```diff
@@ -0,0 +1,277 @@
+import { Injectable } from '@angular/core';
+import { combineLatest, of, Observable, asyncScheduler } from 'rxjs';
+import { filter, map, switchMap, tap, debounceTime } from 'rxjs/operators';
+
+import {
+  Address,
+  CheckoutDeliveryService,
+  UserAddressService,
+  UserPaymentService,
+  PaymentDetails,
+  DeliveryMode,
+  CheckoutPaymentService,
+  LoaderState,
+} from '@spartacus/core';
+import { CheckoutConfigService } from './checkout-config.service';
+import { CheckoutDetailsService } from './checkout-details.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExpressCheckoutService {
+  shippingAddressSet$;
```

Comment:
I'm not sure we want them here, as public API.
And even if, we probably should type them correctly.